### PR TITLE
[build] Add essential PY PKGs on host for sonic-utilities/config/config_mgmt.py

### PR DESCRIPTION
**Pull request by praveen-li**
…config/config_mgmt.py.

Signed-off-by: Praveen Chaudhary pchaudhary@linkedin.com

**- Why I did it**
TO run sonic-utilities/config/config_mgmt.py on host.

**- How I did it**
Add essential PY PKGs on host by adding them in sonic_debian_extension.j2

**- How to verify it**
https://github.com/Azure/sonic-buildimage/pull/4684 was built fine with these changes. [These changes are removed from PR 4684 and then added here.]

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by praveen-li** [comment]
retest vsimage please

**Comment by praveen-li** [comment]
@lguohan it will be helpful if we can merge this PR as well.

File: build_debian.sh [hunk 1]
```diff
@@ -445,6 +445,10 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install 'docker
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install gcc libpython2.7-dev
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install 'netifaces==0.10.7'
 
+# Get package to support Dynamic Port Breakout
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install xmltodict==0.12.0
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install jsondiff==1.2.0
```
Comment (lguohan) [inline build_debian.sh:450]: can you put those dependency in the sonic utilities so that we can automatically install them.

Comment (jleveque) [reply build_debian.sh:450]: Unfortunately, we are currently building sonic-utilities as a .deb package, so it cannot automatically install Python packages which are only available as .whls. We have encountered this limitation with other packages. See https://github.com/Azure/sonic-utilities/blob/master/setup.py#L140

Comment (praveen-li) [reply build_debian.sh:450]: Marking it resolved. Kindly approve and merge.

Comment (jleveque) [reply build_debian.sh:450]: Unresolving to wait for @lguohan to re-review.

Comment (lguohan) [reply build_debian.sh:450]: let's book an item to convert the sonic-utilties into python wheel package?

Comment (jleveque) [reply build_debian.sh:450]: Can do. @lguohan: Do you believe these lines belong here or in sonic_debian_extension.j2?

**Comment by praveen-li** [comment]
@jleveque 

Thanks, Kindly merge it, I need to run tests based on this.

File: build_debian.sh (hunk 1 above)
Comment (praveen-li) [reply build_debian.sh]: Moved to sonic_debian_extension.j2, Marking it resolved. Kindly approve and merge.

File: files/build_templates/sonic_debian_extension.j2 [hunk 2]
```diff
@@ -174,6 +174,10 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install click-d
 # Install tabulate >= 0.8.1 via pip in order to support multi-line row output for sonic-utilities
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install tabulate==0.8.2
 
+# Get package to support Dynamic Port Breakout config command for sonic-utilities
```
Comment (jleveque) [inline files/build_templates/sonic_debian_extension.j2:177]: s/Get package/Install packages/

Comment (praveen-li) [reply files/build_templates/sonic_debian_extension.j2]: @jleveque Fixed.

**Comment by praveen-li** [comment]
retest vs please

**Comment by lguohan** [comment]
retest vsimage please

**Comment by praveen-li** [comment]
@jleveque 
All tests are done, now. We should be good to merge. Thanks.

**Pull request merged**